adb_graphics/figures/maps.py

```diff
@@ -73,7 +75,7 @@ class Map():
                         TILE_DEFS dictionary
     '''
 
-    def __init__(self, airport_fn, ax, **kwargs):
+    def __init__(self, airport_fn, ax, model_name=None, **kwargs):
```

Comment:
It is likely best to include this as a keyword argument if it's not required. An entry in the documentation block above would also be helpful.